.vscode/settings.json

```diff
@@ -19,5 +19,6 @@
     "source.fixAll.eslint": true,
     "source.fixAll.stylelint": true
   },
-  "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
designates workspace version as the ts to use instead of global, solution for [this issue](https://github.com/chakra-ui/chakra-ui/issues/3714)